test/e2e/tests/confirmations/queued-confirmations.spec.ts

```diff
@@ -0,0 +1,423 @@
+/* eslint-disable @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires */
+import { strict as assert } from 'assert';
+import { MockedEndpoint, MockttpServer } from 'mockttp';
+import { getEventPayloads } from '../../helpers';
+import { Driver } from '../../webdriver/driver';
+import { TestSuiteArguments } from './transactions/shared';
+import { MetaMetricsEventName } from '../../../../shared/constants/metametrics';
+
+const FixtureBuilder = require('../../fixture-builder');
+const {
+  withFixtures,
+  openDapp,
+  unlockWallet,
+  DAPP_URL,
+  DAPP_ONE_URL,
+  regularDelayMs,
+  WINDOW_TITLES,
+  defaultGanacheOptions,
+} = require('../../helpers');
+
+describe('Queued Confirmations', function () {
+  if (!process.env.ENABLE_CONFIRMATION_REDESIGN) {
+    return;
+  }
+
+  describe('Existing Requests Banner Alert', function () {
+    it('Banner is shown on dApp 1, but not on dApp 2 after adding transaction on dApp 1, and one on dApp 2', async function () {
+      const port = 8546;
+      const chainId = 1338;
+      await withFixtures(
+        {
+          dapp: true,
+          fixtures: new FixtureBuilder()
+            .withNetworkControllerTripleGanache()
+            .withPreferencesControllerUseRequestQueueEnabled()
+            .withSelectedNetworkControllerPerDomain()
+
+            .build(),
+          dappOptions: { numberOfDapps: 2 },
+          ganacheOptions: {
+            ...defaultGanacheOptions,
+            concurrent: [
+              {
+                port,
+                chainId,
+                ganacheOptions2: defaultGanacheOptions,
+              },
+              {
+                port: 7777,
```

Comment:
The first values are in a constant but not the second, should we hardcode both or create two sets of top level capitalised constants?